pkg/types/types.go

```diff
@@ -339,6 +339,7 @@ type Node struct {
 	Restart       bool              `yaml:"restart" json:"restart,omitempty"`
 	Created       string            `yaml:"created" json:"created,omitempty"`
 	Labels        map[string]string // filled automatically
+	RuntimeLabels map[string]string `yaml:"runtimeLabels" json:"runtimeLabels,omitempty"`
```

Comment:
Those two are basically the same, aren't they?
Since it's going to be a major release, we can replace `Labels` with `RuntimeLabels` (or keep the name and just merge `RuntimeLabels` in without adding it as a new node struct field.

Reply:
Yes you're correct, they're being used for the same thing, however some labels are added automatically by `k3d`, but since user flags don't override them, there shouldn't be an issue to replace it with `RuntimeLabels` to normalize name usage. I will get it fixed.